graphbolt/src/csc_sampling_graph.cc

```diff
@@ -210,7 +220,7 @@ c10::intrusive_ptr<CSCSamplingGraph> CSCSamplingGraph::LoadFromSharedMemory(
   return BuildGraphFromSharedMemoryTensors(std::move(shared_memory_tensors));
 }
 
-torch::Tensor Pick(
+inline torch::Tensor UniformPick(
```

Comment:
Consider also copy the doc string to here as well? Since you already wrote pretty good doc string in the header file.

Reply:
Sure, let me make a copy.